LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1404,7 +1404,7 @@ function getForm(indicatorID, series) {
         }
 
         if(sensitiveChanged){
-            calls.push(            
+            calls.push(
```

Comment:
Guessing all of the below need fail catches.

Reply:
I think this file probably should not even be in this commit. But this is something I have to look at for my code and something to consider for the code around what I am working currently.

Reply:
You are probably right. Looks like the only changes are for spaces. I must have opened it and made a small change then changed it back or something and the space changes stuck. I'm sure VSCode removed the extra spaces when I saved.